File: examples/python/quick_start.py
from e3d_client import E3DClient


def main() -> None:
    client = E3DClient()
    openapi = client.get("/openapi")
    rate = client.get("/rate")

    title = (openapi or {}).get("info", {}).get("title", "unknown")
    print(f"OpenAPI title: {title}")
    print(f"Current rate: {rate.get('rate') if isinstance(rate, dict) else rate}")


if __name__ == "__main__":
    main()

File: examples/quick-start.mjs
import { E3D } from '../dist/index.js';

const e3d = new E3D({
  baseUrl: process.env.E3D_BASE_URL || 'https://e3d.ai/apitest',
  apiKey: process.env.E3D_API_KEY,
});

const openApi = await e3d.discovery.getOpenApi();
const rate = await e3d.discovery.getRate();

console.log('OpenAPI title:', openApi.info?.title || 'unknown');
console.log('Current rate:', rate.rate);
